index.html

```diff
@@ -1,49 +1,189 @@
+    <title>Prangon's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="logo-placeholder">
+                <a href="#" class="my-logo">PG</a>
+            </div>
+            <div class="menu">
+                <div class="icon">
+                    <nav class="union">
+                        <img src="images/Union.png" alt="menu-image" class="menu-image">
+                    </nav>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section class="container-headline">
+        <div class="headline">
+            <h1 class="primary-text">
+                I'm Prangon<br>Glad to see you!
+            </h1>
+            <p class="description">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <div class="social-media">
+                <h2 id="media-text">
+                    LET'S CONNECT
+                </h2>
+                <ul class="media-icons">
+                    <li class="icon-list"><a href="#"><img src="images/Twitter.png" alt="Twitter" id="twitter"></a></li>
+                    <li class="icon-list"><a href="#"><img src="images/Linkedin.png" alt="Linkedin" id="linkedin"></a></li>
+                    <li class="icon-list"><a href="#"><img src="images/M.png" alt="M" id="M"></a></li>
+                    <li class="icon-list"><a href="#"><img src="images/GitHub.png" alt="GitHub" id="github"></a></li>
+                    <li class="icon-list"><a href="#"><img src="images/Victory.png" alt="Victory" id="victory"></a></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <div class="project">
+        <section class="tonic-1">
+            <img class="photo-grid" src="images\Tonic1.svg" alt="Project picture">
+            <h1 class="project-title">Tonic</h1>
+            <section class="canopy-segment">
+                <h2 class="canopy">CANOPY</h2>
+                <ul class="canopy-items">
+                    <li><img src="images\Counter.svg"></li>
+                    <li class="item">Back End Dev</li>
+                    <li><img src="images\Counter.svg"></li>
+                    <li class="item">2022</li>
+                </ul>
+            </section>
+            <p class="project-description">
+                A daily selection of privately personalized 
+                reads; no accounts or sign-ups required.
+            </p>
+            <ul class="coding-languages">
+                <li class="item-languages">html</li>
+                <li class="item-languages">css</li>
+                <li class="item-languages">javaScript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+        </section>

```

Comment:
- Hey, here's an improvement you can do for each card. You are meant to align each card using flexbox according to the project requirement. There are many ways to achieve that. According to your layout, it's not structured correctly to help achieve this aim, as you're using and nesting many sections in another section and this is not a good practise in using the semantic tag correctly. below is a structure you can use to achieve your aim:

``` 
    <div class="project-work">

        <div class="prjects tonic-1">
            <div class="image-container">
                <img class="photo-grid" src="images\Tonic1.svg" alt="Project pictre">
            </div>
            <div class="project-body">
            <h1 class="project-title">Tonic</h1>
            <div class="canopy-segment">
                <span class="">CANOPY</span>
                <img src="images\Counter.svg">
                <span class="item">Back End Dev</span>
                <img src="images\Counter.svg">
                <span class="item">2022</span>
            </div>
            <p class="project-description">
                A daily selection of privately personalized reads; no accounts or sign-ups required.
            </p>
            <ul class="coding-languages">
                <li class="item-languages">html</li>
                <li class="item-languages">css</li>
                <li class="item-languages">javaScript</li>
            </ul>
            <button type="button" class="see-project">See Project</button>
            </div>
        </div>
        duplicate the card
        duplicate the card
        duplicate the card
        
    </div>
```

---

- I had to restructure your layout here to help you fix this issue
- Each card that represents one project can just use the same CSS rule as all cards have the same structure in your case, so it helps you in fixing the layout.
- You should be able to adjust your styling from here using flexbox and not grid for each card.